src/frontend/components/SelectContent/index.tsx

```diff
@@ -421,6 +433,23 @@ export const SelectContent = ({
       </form>
 
       <Tabs>
+        <Tab title="Webinars">
+          <SelectContentSection
+            addMessage={messages.addWebinar}
+            addAndSelectContent={async () => {
+              useCreateVideoMutation.mutate({
+                playlist: playlist!.id,
+                title: lti_select_form_data?.activity_title,
+                description: lti_select_form_data?.activity_description,
+                live_type: LiveModeType.JITSI,
+              });
+            }}
+            newLtiUrl={new_video_url!}
+            items={webinars!}
+            selectContent={selectContent}
```

Comment:
And those two ?

Reply:
Outside of this PR too (as all resources types should be treated in the same way), but yes!